src/renderer/views/Playlist/Playlist.js

```diff
@@ -679,6 +685,16 @@ export default defineComponent({
       }
     },
 
+    formatDuration(totalSeconds, locale = 'en') {
+      const duration = {
+        hours: Math.floor(totalSeconds / 3600),
+        minutes: Math.floor((totalSeconds % 3600) / 60),
+        seconds: totalSeconds % 60,
+      }
+
+      return new Intl.DurationFormat(locale, { style: 'short' }).format(duration)
```

Comment:
This should use the user's locale and fallback to english
```suggestion
      return new Intl.DurationFormat([this.currentLocale, 'en'], { style: 'short' }).format(duration)
```